CONTRIBUTING.md

```diff
@@ -167,13 +167,11 @@ If using `docker buildx`, run:
 docker buildx build -t ibeam --load .
 ```
 
-Alternatively, `docker-compose` can be used to build and run a local IBeam instance as follows:
+Alternatively, `docker compose` can be used to build and run a local IBeam instance as follows:
 
-Create a `docker-compose.yml` file with the following content:
+Create a `compose.yaml` file with the following content:
 
 ```yaml
-version: "2.1"
```

Comment:
I don't think we should remove the version

Reply:
docker compose v2 ignore top level version tag